src/supabase-to-zod.ts

```diff
@@ -49,11 +49,15 @@ export default async function supabaseToZod(opts: SupabaseToZodOptions) {
 
   const parsedTypes = transformTypes({ sourceText, ...opts });
 
-  const { getZodSchemasFile } = generate({
+  const { getZodSchemasFile, errors } = generate({
     sourceText: parsedTypes,
     ...opts,
   });
 
+  if (errors.length > 0) {
+    throw new Error(errors.join('\n'));
```

Comment:
Let me know if you find this behavior undesirable, but I think it is reasonable to fail fast and save the users confusion down the line in case they find some types are missing.